integration/spark/src/main/scala/org/apache/spark/sql/optimizer/CarbonFilters.scala

```diff
@@ -286,6 +287,11 @@ object CarbonFilters {
         val (columnName, instance) = getGeoHashHandler(relation.carbonTable)
         Some(new PolygonRangeListExpression(children.head.toString(), children.last.toString(),
           columnName, instance))
+      case _: BlockPathsUDF =>
+        if (children.size > 1) {
+          throw new MalformedCarbonCommandException("Expect one string in polygon")
```

Comment:
exception message is not proper, please fix

Reply:
yeah, updated now